rss_connector.py

```diff
@@ -100,10 +100,7 @@ def _cmp_with_last_checked_entry(self, entry, last_checked_entry):
         return False
 
     def _save_html(self, html_file, name, container_id):
-        if hasattr(Vault, 'get_vault_tmp_dir'):
-            fd, path = tempfile.mkstemp(dir=Vault.get_vault_tmp_dir(), text=True)
-        else:
-            fd, path = tempfile.mkstemp(dir='/opt/phantom/vault/tmp', text=True)
+        fd, path = tempfile.mkstemp(dir=Vault.get_vault_tmp_dir(), text=True)
```

Comment:
`Vault.get_vault_tmp_dir()` is available in Phantom 4.9 (or maybe even earlier) and current version of connector has  `"min_phantom_version": "5.5.0"` so I think some of this dead code could have been cleaned up.